Source/Classes/AnimatedImages/PINGIFAnimatedImage.m

```diff
@@ -79,7 +79,7 @@ + (Float32)frameDurationAtIndex:(NSUInteger)index source:(CGImageSourceRef)sourc
         }
     }
     
-    if (frameDuration < kPINAnimatedImageMinimumDuration) {
+    if (frameDuration < (1.0 / [PINAnimatedImage maximumFramesPerSecond])) {
```

Comment:
The `1.0 / [PINAnimatedImage maximumFramesPerSecond]` value could be cached here and below.